elements/pfe-select/package.json

```diff
@@ -19,7 +19,7 @@
     "build": "../../node_modules/.bin/gulp && ../../node_modules/.bin/prettier --ignore-path ../../.prettierignore --write '**/*.{js,json}'",
     "dev": "../../node_modules/.bin/gulp dev",
     "watch": "../../node_modules/.bin/gulp watch",
-    "test": "../../node_modules/.bin/wct --configFile ../../wct.conf.json elements/pfe-select/test/"
+    "test": "echo \"👉 Run tests from the project root with 'npm run test:watch -- elements/*/test/pfe-select.spec.js'\""
```

Comment:
cc @kylebuch8 for your thoughts on this update ^

Reply:
The long term goal would be for individual elements to have the right dependencies to run their own tests.